### ISSUE TITLE
Jikkou cli issue with glibc for jikkou v0.28.0 - v0.35.0 on RHEL-8 

### ISSUE THREAD

**Issue by obiremiv** [streamthoughts/jikkou#502]
I installed jikkou with sdk for ~>0.30 and also went down a bit to v0.28.0 but it requires >glibc-2.29.

OS: `RHEL-8`
glibc version:  glibc-2.28-251.el8_10.5.x86_64

Please can you share the link to the dependencies required for jikkou where i can get the list of requirements for various linux distributions. If possible please help me with dependent glibc installation for RHEL-8.. Thanks.

```
[root@cnflntbrkrsb1s1 bin]# ./jikkou
./jikkou: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./jikkou)
./jikkou: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./jikkou)
./jikkou: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./jikkou)
```

**Comment by fhussonnois** [comment]
Hi @obiremiv, really sorry for that error first time I see it with Jikkou. Jikkou native build should only depends on GLibC (no other dependency). If you struggle or cannot get the required version for that then I recommand to use the java binary: https://github.com/streamthoughts/jikkou/releases/tag/v0.35.5

**Comment by obiremiv** [comment]
@fhussonnois yeah that didnt work had to pull from the hub and place this in a .bashrc file cause it was dissappearing after the entry

```
# alias jikkou='podman run -it --network host -v /opt/jikkou/jikkou/config:/etc/jikkou/config -v /opt/jikkou/jikkou/application.conf:/etc/jikkou/application.conf streamthoughts/jikkou:0.35.5'
 
 and 
 
# source ~/.bashrc 
```

**Comment by fhussonnois** [comment]
Hi @obiremiv, in the JSON jikkou config file, you should not forget to configure the `configFile` property to specify the location of the config file for the context to be used. Something like : 

```json
{
  "currentContext": "localhost",
  "localhost": {
    "configFile": "/etc/jikkou/application.conf",
    "configProps": {
      ...
    }
  }
}
```

**Comment by obiremiv** [comment]
Thanks I missed that, but I tried and it loaded the security protocol but it's failing to load the keystore because of access which understandable because the user is the running process which is cp-kafka AND also because I specified the path where the keystore is just like it was specified in the client.properties file on the server. It's failing to load the keystore

Question
Do I have to mount the keystore as well if so that requires changing the user

Also the Kafka we have is confluent-Kafka on premise which is the platform does jikkou interact with confluent-Kafka on premise(confluent-platform) or it only work seamlessly with SASL ON CONFLUENT CLOUD

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by obiremiv** [comment]
Thanks Florian got it to work needed to run the container/image with the id of the user:group of the certificate...Thanks for all your help.

________________________________
